pyproject.toml

```diff
@@ -29,7 +29,7 @@ keywords = [
 ]
 dynamic = ['version']
 dependencies = [
-    "attrs>=22.2.0",
+    "attrs>=24.1.0",
```

Comment:
I have no objections, but this PR should only be merged once the new attrs version is available on pip right?

Reply:
correct 👍🏼